File: whale/src/main/java/com/flannery/whale/SecondActivity.java
package com.flannery.whale;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        MyApp.listActivity.add(this);

        findViewById(R.id.finishThis).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: projects_/LoggerProject/app/src/main/java/com/joyy/loggerproject/SecondActivity.kt
package com.joyy.loggerproject

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.joyy.log_core.KLog

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        KLog.et()
    }
}